MdePkg/MdePkg.dec

```diff
@@ -104,7 +104,7 @@
 
 ## MU_CHANGE [BEGIN]
   ##  @libraryclass  Provides a hook called when a stack cookie check fails.
-  StackCheckFailureLib|Include/Library/StackCheckFailureLib.h
+  StackCheckFailureHookLibNull|Include/Library/StackCheckFailureHook.h

```

Comment:
This should be the library class name, not the library class instance name - i.e.  `StackCheckFailureHookLib`